src/viam/ur/module/ur_arm.cpp

```diff
@@ -948,13 +945,25 @@ URArm::UrDriverStatus URArm::read_joint_keep_alive_(bool log) {
         // reset the driver client so we stop trying to ask for more data
         current_state_->driver->resetRTDEClient(current_state_->appdir + k_output_recipe, current_state_->appdir + k_input_recipe);
 
+        // delay so we don't spam the dashboard client if disconnected
+        std::this_thread::sleep_for(k_estop_delay);
+
+        // start capturing data from the arm driver again
+        current_state_->driver->startRTDECommunication();
+
         VIAM_SDK_LOG(error) << "failed to talk to the arm, is the tablet in local mode? : " << std::string(ex.what());
-        return UrDriverStatus::DASHBOARD_FAILURE;
     }
 
-    if (status.find(urcl::safetyStatusString(urcl::SafetyStatus::NORMAL)) == std::string::npos) {
-        // the arm is currently estopped. save this state.
-        current_state_->estop.store(true);
+    // check if the arm status is normal or not empty. the status will be empty if the arm is disconnected or when the arm is first switched
+    // into local mode
+    if ((status.find(urcl::safetyStatusString(urcl::SafetyStatus::NORMAL)) == std::string::npos) && !status.empty()) {
+        VIAM_SDK_LOG(info) << "read_joint_keep_alive dashboard->commandSafetyStatus() arm status : " << status;
+        // the arm is currently estopped. if the user is not in local mode, save this state.
+        // if the user is in local mode, then we assume that they opted into this state.
+        // the UR20 will always return a stop while in manual mode, unless the three position enabling button is held down
+        if (!current_state_->local_disconnect.load()) {
+            current_state_->estop.store(true);
+        }
```

Comment:
also not a huge fan of this. the other "easy" option is to always run through the estop recovery if an estop is detected in local mode. 

the issue I had with that was the ur20 basically always reports an estop when switching into local mode, and that could get pretty tedious for the motion team while testing